pipeline/terraform/modules/stack/service_id_minter.tf

```diff
@@ -45,15 +45,8 @@ module "id_minter" {
   cpu    = 2048
   memory = 4096
 
-  # The total number of connections to RDS across all tasks from all ID minter
-  # services must not exceed the maximum supported by the RDS instance.
   min_capacity = var.min_capacity
-  max_capacity = min(
-    floor(
-      local.id_minter_rds_max_connections / local.id_minter_task_max_connections
-    ),
-    local.max_capacity
-  )
+  max_capacity = local.max_capacity
```

Comment:
The fiddling with max_capacity here was to account for the scale of the database, the Aurora serverless stuff now means that it will scale with demand up to about ~15 id_minter tasks. `local.max_capacity` is capped at 12, so hopefully we'll not see a problem with overwhelming RDS.